packages/insured-bridge-relayer/src/canonical-bridge-adapters/BridgeAdapterInterface.ts

```diff
@@ -0,0 +1,8 @@
+import type { TransactionType } from "@uma/common";
+
+export default interface ExchangeAdapterInterface {
+  initialize(): void;
+  constructCrossDomainFinalizationTransaction(
+    transactionHash: string
+  ): Promise<{ l2TransactionHash: string; finalizationTransaction: TransactionType | null }>;
```

Comment:
why `| null`?

Reply:
if the relay cant be sent (it's not confirmed yet) it is null! this is the key mechanism on how filtering works :)